### ISSUE TITLE
Add focus indicator for header Search input

### ISSUE THREAD

**Issue by BlueSlug** [inclusive-design/wecount.inclusivedesign.ca#651]
## Is your feature request related to a problem? Please describe.

The "search" input in the site header does not have a visible focus indicator.

## Describe the solution you'd like

A design has been created by @uttaraghodke:
![image](https://user-images.githubusercontent.com/547933/125369604-8e55ba00-e34a-11eb-875a-ae2c71bd4712.png)

## Describe alternatives you've considered

N/A

## Additional context

The design describes a pair of rounded borders, which adds some complexity to the solution. The `outline` CSS property is not appropriate since it does not account for the rounded border of the element.
